src/windows/functions.cpp

```diff
@@ -393,6 +393,12 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterValid(TCHAR driveLetter) {
+  int driveLetterIndex = static_cast<int>(driveLetter) - static_cast<int>('A');
```

Comment:
> Make sure you also test (if you can) having a network drive as e.g. E: and then try ejecting a removable drive at G:

You're a genius. This triggers the same problematic error despite the fix :tada: I'll fix this one as well here.